core/src/main/java/org/mini2dx/Tappybird/InputHandler.java

```diff
@@ -0,0 +1,18 @@
+package org.mini2dx.Tappybird;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+
+public class InputHandler {
+
+    //TODO Maybe add an abstraction here to the player input.
+    boolean SpacePressed () {
```

Comment:
Method names in Java use the following style:
isSpacePressed()
goToSomewhere()